test/src/system/pasting_test.coffee

```diff
@@ -85,6 +85,13 @@ testGroup "Pasting", template: "editor_empty", ->
       assert.textAttributes([0, 7], href: "http://example.com")
       expectDocument "Example\n"
 
+  test "paste JavaScript URL", (expectDocument) ->
+   pasteData =
+     "URL": "javascript:alert('XSS')\n"
```

Comment:
Is the `\n` in this string significant? Let's remove it if not.

Reply:
It is not significant or important. I will remove it.